src/utils/useUpdateEffect/useUpdateEffect.ts

```diff
@@ -0,0 +1,43 @@
+import * as React from 'react';
+
+// useUpdateEffect is a useEffect that doesn't fire on initial render
+// for detecting when values change
+// https://stackoverflow.com/a/58217148
+
+const useIsMounted = function useIsMounted() {
+    const isMounted = React.useRef(false);
+
+    React.useEffect(function setIsMounted() {
+        isMounted.current = true;
+
+        return function cleanupSetIsMounted() {
+            isMounted.current = false;
+        };
+    }, []);
+
+    return isMounted;
+};
+
+export const useUpdateEffect = function useUpdateEffect(
+    effect: () => void | (() => void),
+    dependencies: Array<any>
```

Comment:
eslint: `Array type using 'Array<any>' is forbidden. Use 'any[]' instead`

Reply:
Lol, I tried `Any[]` but couldn't remember the right syntax. Ty